### DISCUSSION TITLE
teoria relazionale - 6

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#73] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/f6619180-e9e4-4630-927d-8233dcca2f18)

**Comment by Carlo-Armato** [comment]
![Cattura](https://github.com/sapienzastudentsnetwork/bdi2324/assets/158755407/a8df60c3-5b0a-473e-9dfb-0198fe607622)

**Reply by Frisbon** [reply]
L'ultimo passaggio, hai scelto $ACD, ABDE$ perché uguali su $DG$ ma su $EB$... **non** sono diversi, quindi non potresti applicare il passaggio.

(NON É VERO)

**Reply by Carlo-Armato** [reply]
b1,b1 non è diverso da a,a?

**Reply by Frisbon** [reply]
Lascia stare, sono le 6 ore di esercizi che mi fanno leggere in colonna....
Sorry il disturbo

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/95f8d7fc-c497-4b76-a3d5-7b239cfa22a1)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/52c80786-a800-492c-848c-cd1ab4823327)
